templates/robot_description/view_robot.launch.py

```diff
@@ -19,81 +19,31 @@
 #
 
 from launch import LaunchDescription
-from launch.actions import DeclareLaunchArgument
-from launch.substitutions import Command, FindExecutable, LaunchConfiguration, PathJoinSubstitution
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.actions import IncludeLaunchDescription
+from launch.substitutions import PathJoinSubstitution
 from launch_ros.actions import Node
 from launch_ros.substitutions import FindPackageShare
 
 
 def generate_launch_description():
-    # Declare arguments
-    declared_arguments = []
-    declared_arguments.append(
-        DeclareLaunchArgument(
-            "description_package",
-            default_value="$PKG_NAME$",
-            description="Description package of the $ROBOT_NAME$. Usually the argument is not set, \
-        it enables use of a custom description.",
-        )
-    )
-    declared_arguments.append(
-        DeclareLaunchArgument(
-            "prefix",
-            default_value='""',
-            description="Prefix of the joint names, useful for \
-        multi-robot setup. If changed than also joint names in the controllers' configuration \
-        have to be updated.",
-        )
-    )
-
-    # Initialize Arguments
-    description_package = LaunchConfiguration("description_package")
-    prefix = LaunchConfiguration("prefix")
-
-    # Get URDF via xacro
-    robot_description_content = Command(
-        [
-            PathJoinSubstitution([FindExecutable(name="xacro")]),
-            " ",
+    robot_description_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
             PathJoinSubstitution(
-                [FindPackageShare(description_package), "urdf", "$ROBOT_NAME$.urdf.xacro"]
-            ),
-            " ",
-            "prefix:=",
-            prefix,
-            " ",
-        ]
-    )
-
-    robot_description = {"robot_description": robot_description_content}
-
-    rviz_config_file = PathJoinSubstitution(
-        [FindPackageShare(description_package), "rviz", "$ROBOT_NAME$.rviz"]
+                [FindPackageShare("$PKG_NAME$"), "launch", "$ROBOT_NAME$.launch.py"]
```

Comment:
With this fix the right launch file was found:
```suggestion
                [FindPackageShare("$PKG_NAME$"), "launch", "$ROBOT_NAME$_description.launch.py"]
```